script.js

```diff
@@ -121,4 +123,33 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
     error.innerText = 'Please Your Email Must be in Lower Case';
   }
-});
\ No newline at end of file
+});
+// Populate LocalStrorage
+function populateStorage() {
+  formObj = {
+    Name: form.fullname.value,
+    Email: form.email.value,
+    Message: form.message.value,
+  };
+  const convertObj = JSON.stringify(formObj);
+  localStorage.setItem('FormData', convertObj);
+}
```

Comment:
As required this project let's fix the `populateStorage` method currently it's setting the data to `local storage` when the user clicks and submits the form. But as stated in the project requirement it should have to set the data to `local storage` when the user changes any of the input fields 👇 .
<img width="956" alt="image" src="https://user-images.githubusercontent.com/34578631/168036897-0a85f6db-2a1c-4a9a-b3b9-552e92add164.png">

If I fill out the form and before submitting it if I refresh the page I lose all the data I entered into the input field. Kindly, fix this to persist the data on page refresh even before submitting the form.

Reply:
- [ ] Kindly check on your form input persistency as instructed by the reviewer, currently the input is lost on page reload.